packages/core/upload/tests/content-api/upload.test.api.js

```diff
@@ -163,6 +164,33 @@ describe('Upload plugin', () => {
         ])
       );
     });
+    test('Get one file', async () => {
+      const dogEntity = await strapi.entityService.create('api::dog.dog', {
+        data: {},
+        files: {
+          profilePicture: {
+            path: path.join(__dirname, '../utils/rec.jpg'),
+            name: 'rec',
+            type: 'jpg',
+            size: 0,
+          },
+        },
+        populate: 'profilePicture',
+      });
+      const getRes = await rq({
+        method: 'GET',
+        url: `/upload/files/${dogEntity.profilePicture.id}`,
+      });
+
+      expect(getRes.statusCode).toBe(200);
+      expect(getRes.body).toEqual(
+        expect.objectContaining({
+          id: expect.anything(),
+          url: expect.any(String),
+        })
+      );
+      await strapi.entityService.delete('api::dog.dog', dogEntity.id);
```

Comment:
is it necessary to delete dog entity ? 
The file entity will not be deleted with this I think.

Reply:
That is true, I should also delete the file.

It's a good practice to remove everything that is useless after the test.
I would have done it in the "beforeAll" by default, but as this entity isn't used anywhere else in this describe section, it would not make sense.